libpod/container.go

```diff
@@ -1044,3 +1057,14 @@ func (c *Container) cleanupStorage() error {
 
 	return c.save()
 }
+
+// copyRresolveToRunDir copies /etc/resolv.conf to the containers
+// rundir so it can be mounted into the container for DNS
+// only call this is you already have a lock
+func (c *Container) copyResolvToRunDir() error {
```

Comment:
Suggestion: you can have this return (string, error) with the string being runDirResolv - that way you don't have to have a separate filepath.Join() in Init()

Reply:
++ like that one